### ISSUE TITLE
Accessibility for Main Navigation: Ensure ARIA regions, landmarks and HTML sections are identifiable

### ISSUE THREAD

**Issue by cjwest** [SU-SWS/decanter#395]
**Accessibility for Main Navigation**

***Ensure ARIA regions, landmarks and HTML sections are identifiable (Severity: 3)***
[Issue]
Each instance of the menu has the same aria-label. (Note - the tester observed guidance in the documentation that said unique labels should be used in real use, but still notes it here for the record).

[User Impact]
Screen reader users will not be able to tell the menus apart without unique labels.

[Code Reference]
<nav class="su-main-nav" aria-label="main menu" style="z-index: 9699;" aria-expanded="false">

<nav class="su-main-nav su-main-nav--light" aria-label="main menu" style="z-index: 9399;" aria-expanded="false">

etc.

**To Reproduce**
1. Go to http://decanter.stanford.edu/section-components.html#kssref-components-main-navigation

**Expected behavior**
Expected Behavior]
Ensure when there are multiple menus on a page that unique labels are provided for each one.

[Code Example]
<nav class="su-main-nav" aria-label="main menu" style="z-index: 9699;" aria-expanded="false">

<nav class="su-main-nav su-main-nav--light" aria-label="secondary menu" style="z-index: 9399;" aria-expanded="false">

etc.

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
https://stanford.levelaccess.net/public/reporting/view_module.php?module_id=9631555
